Production_scripts/convert_gp_costs.R

```diff
@@ -0,0 +1,143 @@
+#####################################################
+# Costs - GP Out of Hours
+# Author: Catherine Holland
+# Date: February 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input -
+# Description -
+#####################################################
+
+
+library(dplyr)
+library(ggplot2)
+
+
+## Make a copy of the existing file ##
+# read data in
+current_file <- haven::read_sav(get_gp_ooh_costs_path())
+
+
+# write data to folder
+# .zsav
+haven::write_sav(current_file,
+  paste0(get_slf_dir(), "/Costs/Cost_GPOoH_Lookup_pre", latest_update(), ".zsav"),
+  compress = TRUE
+)
+
+
+
```

Comment:
Can you remove a lot of this white space please? I think one blank line between functions and maybe 2 between 'sections' is enough.